contents/handbook/engineering/support-hero.md

```diff
@@ -4,35 +4,59 @@ sidebar: Handbook
-- [Zendesk](https://posthoghelp.zendesk.com/agent/filters/5586845866651) - look for the dedicated folder for your team. If new tickets are created, then a slack notification will be sent also to your team's dedicated support channel.
-- [PostHog Users' Slack](/slack), specifically `#community` and `#general` or elsewhere should be redirected to using [the bug button](https://app.posthog.com/home#supportModal) within the app, which provides us with all the context and helps triage. We do not commit to providing support through Slack, and should suggest users who prefer this channel ask @Max-AI as a first port of call.
-- Sentry issues, either [directly](https://sentry.io/organizations/posthog/issues/?project=1899813) or in `#sentry` in our main Slack.
-- [Community forums](/questions) - each small team can decide how to handle questions pertaining to their part of the product, but we suggest working with your small team's marketing representative to make sure each question gets answered. (Read more in the [Community](/handbook/small-teams/website-docs/community) section of the handbook.)
+Swap with a teammate in advance! Find a volunteer by asking in Slack, then use PagerDuty schedule overrides. You can trade whole weeks, but also just specific days. Remember not to alter the rotation's core order, as that's an easy way to accidentally shift the schedule for everyone.
 
-### Communication
+## What do I do as Support Hero?
+
+Each engineering team has its own list of tickets in Zendesk:
+
+- [Product Analytics](https://posthoghelp.zendesk.com/agent/filters/17989255082139) (escalated tickets only)
+- [Feature Success](https://posthoghelp.zendesk.com/agent/filters/14507099125531)
+- [Replay](https://posthoghelp.zendesk.com/agent/filters/14507048415771)
+- [Pipeline](https://posthoghelp.zendesk.com/agent/filters/14506794017051)
+- [Infrastructure](https://posthoghelp.zendesk.com/agent/filters/14507148758939)
+- [Auth & Billing, handled by Growth](https://posthoghelp.zendesk.com/agent/filters/14507107058843)
+
+Your job is simple: resolve ticket after ticket from your team's list.
+
+There are three sources of tickets:
+
+1. In-app bug reports/feedback/support tickets sent from the [Support panel](https://us.posthog.com/home#panel=support). They include a bunch of useful links, e.g. to the admin panel and to the relevant session recording.
+1. Community questions asked on PostHog.com.
+1. Slack threads that have been marked with the 🎫 reaction in customer support channels.
+
+Some of them will ask for new features. If the feature would be widely useful for users matching [our ICP](/handbook/who-we-are-building-for), it might be a good idea to build it. Otherwise, feel free to just create a feature request issue in GitHub or +1 on an existing one – you can then send a link to the user, giving them a way of tracking progress.
```

Comment:
You can't +1 twice unless you add comments which is not ideal.
what's the current status quo for feature requests? how would you know if 1 or 10 customers asked for it via tickets instead of +1 themselves?

Reply:
I think comments are the way, here's one way to put this:
```suggestion
Some of them will ask for new features. If the feature would be widely useful for users matching [our ICP](/handbook/who-we-are-building-for), it might be a good idea to build it. Otherwise, feel free to just create a feature request issue in GitHub or, if one already exists, comment with "+1 from <ticket link>" – you can then send an issue link to the user, giving them a way of tracking progress.
```

Reply:
I'm not a big fan of +1 comments, you still have to count them every time you wanna know how many people asked for it and every person subscribed to the issugetset alerted, but if we rely on GH only, I don't know a better way.

Reply:
We are also tracking feature requests in Vitally.

```suggestion
Some of them will ask for new features. If the feature would be widely useful for users matching [our ICP](/handbook/who-we-are-building-for), it might be a good idea to build it. Otherwise, feel free to just create a feature request issue in GitHub or +1 on an existing one – you can then send a link to the user, giving them a way of tracking progress. Also make sure to let the [Customer Success Team](https://posthog.com/handbook/small-teams/customer-success) know, since they will track feature requests for paying customers.
```

cc: @simfish85